frontend/test/cypress/integration/product-list/parts_page_devices.spec.ts

```diff
@@ -0,0 +1,84 @@
+describe('parts page devices', () => {
+   const user = cy;
+
+   beforeEach(() => {
+      user.visit('/Parts');
+   });
+
+   it('should visit all devices on /Parts page', () => {
+      assertVisibleFilterAndProducts();
+      assertAvailableProducts();
+
+      const visitPageCount = 4;
+      user
+         .getBySel('product-list-devices')
+         .find('a')
+         .each((device, index) => {
+            // Only visit few device pages since it will take too long to visit all
+            if (index == visitPageCount)
+               return false;
+
+            let deviceLink = device.attr('href');
+            if (deviceLink) {

```

Comment:
I had to do this (workaround for typescript) because `device.attr('href')` returned `string | undefined`:
![devicelink](https://user-images.githubusercontent.com/58952979/177663958-b087eca5-185d-4e69-8a20-1113fd11a5d6.jpg)
 

but since we're dropping this test, we don't need it anymore.